docs/network/transfers.md

```diff
@@ -135,23 +135,16 @@ Transactions are signed using the [EIP-712](https://github.com/ethereum/EIPs/blo
 }
 ```
 
-Note, the child chain server collects fees for sending a transaction. The fee can be paid in a variety of supported tokens by the network. To get more details on how the fees are defined, please refer to [Fees](https://docs.omg.network/fees).
+The child chain server collects fees for sending a transaction. The fee can be paid in a variety of supported tokens by the network, the fee amount is automatically calculated by the child chain. To get more details on how the fees are defined, please refer to [Fees](/network/fees). Note, testnet and mainnet might support different currencies for fees.
 
 #### 3.1 Method A
 
-The most "granular" implementation of transfer includes fetching fees, creating, typing, signing and submitting the transaction. Such an approach will have the following structure of the code:
+The most "granular" implementation of transfer includes creating, typing, signing and submitting the transaction. Such an approach will have the following structure of the code:
```

Comment:
referring to my comment above, the below example here can actually be even more "granular" by forming the transaction locally, which would require fetching the fees. but for the current example used, this is correct that you dont need the fee amount

Reply:
What is the method/workflow for forming a transaction locally? Did we cover it somewhere? If not, it would be great to add it as a Method C : )

Reply:
https://github.com/omgnetwork/omg-js/blob/master/packages/integration-tests/test/createTransactionTest.js#L345

you can see here, its more or less the same, its just creating the transaction is done in the lib, and not with the watcher